cmd/cloudx/project/get.go

```diff
@@ -45,7 +45,15 @@ $ ory get project ecaaa3cb-0730-4ee8-a6df-9553cdfeef89 --format json
 				return err
 			}
 
-			project, err := h.GetProject(args[0])
+			var id string
+			if len(args) == 0 {
+				if id, err = h.GetDefaultProjectID(); err != nil {
+					return cmdx.PrintOpenAPIError(cmd, err)
+				}
+			} else {
+				id = args[0]
+			}
+			project, err := h.GetProject(id)
```

Comment:
This section is now repeated across all commands that previously took a project ID as part of the args. Can we just put this into `GetDefaultProjectID` as well?

```suggestion
			project, err := h.GetDefaultProjectID(args)
```

Reply:
done